javascript/schema/src/databundle/DataBundleSchema.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { CodebookSchema } from '../codebook/CodebookSchema.js'
+import type { ResourceSchema } from '../datapackage/ResourceSchema.js'
+import type { DataTableSchema } from '../datatable/DataTableSchema.js'
+import type { WorkflowSchema } from '../workflow/WorkflowSchema.js'
+
+/**
+ * This defines the data-bundle schema, which is a collection of table-centric resources.
+ * The data-table represents the data-source and parsing options.
+ * The codebook represents interpretation options and data-type mappings.
+ * The workflow represents data-wrangling steps.
+ */
+export interface DataBundleSchema extends ResourceSchema {
+	/**
+	 * Resource path to the datatable file, which contains the data source and parsing options.
+	 */
+	datatable?: string | DataTableSchema
```

Comment:
I don't think this is how we discussed the organization. Weren't we going to remove strongly-typed subschemas (the main one being codebook baked into DataTable I believe), and rely on resource handlers to understand how to apply their sources?

I thought the specific fixed types were going to end up as resource view config field wells, rather than on core schema